charts/nri-bundle/README.md

```diff
@@ -98,81 +55,72 @@ In the following example you can see how to monitor a Redis integration with aut
-In order to update any of the dependency versions you should bump the version in `requirements.yaml` and run `helm dependency update` 
-command to update chart packages under `/charts` and also the `requirements.lock` file.
-
-ATTENTION: do not commit the internal dependencies. We only commit *kube-state-metrics* and `pixie-chart` because we 
-don't own their distribution but would like to be resilient to failures of their repository.
-
-## Example
-
-Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
-
-Then, to install this chart, run the following command:
-
-```sh
-helm install newrelic/nri-bundle \
---set global.licenseKey=<New Relic License key> \
---set global.cluster=<Cluster name> \
---set infrastructure.enabled=true \
---set prometheus.enabled=true \
---set webhook.enabled=true \
---set ksm.enabled=true \
---set kubeEvents.enabled=true \
---set logging.enabled=true \
---generate-name
-```
-
-### Example with Pixie
-
-```sh
-helm install newrelic/nri-bundle \
---set global.licenseKey=<New Relic License key> \
---set global.cluster=<Cluster name> \
---set infrastructure.enabled=true \
---set prometheus.enabled=true \
---set webhook.enabled=true \
---set ksm.enabled=true \
---set kubeEvents.enabled=true \
---set logging.enabled=true \
---set newrelic-pixie.enabled=true \
---set newrelic-pixie.apiKey=<Pixie API key> \
---set pixie-chart.enabled=true \
---set pixie-chart.deployKey=<Pixie Deploy key> \
---set pixie-chart.clusterName=<Cluster name> \ 
---generate-name
+  nri-redis-sampleapp:
+    discovery:
+      command:
+        exec: /var/db/newrelic-infra/nri-discovery-kubernetes --tls --port 10250
+        match:
+          label.app: sampleapp
+    integrations:
+      - name: nri-redis
+        env:
+          # using the discovered IP as the hostname address
+          HOSTNAME: ${discovery.ip}
+          PORT: 6379
+        labels:
+          env: test
 ```
 
-[1]: https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key
-[2]: https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics
-[3]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure
-[4]: https://github.com/newrelic/helm-charts/tree/master/charts/nri-prometheus
-[5]: https://github.com/newrelic/helm-charts/tree/master/charts/nri-metadata-injection
-[6]: https://github.com/newrelic/helm-charts/tree/master/charts/nri-kube-events
-[7]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging
-[8]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie
-[9]: https://docs.pixielabs.ai/installing-pixie/install-schemes/helm/#3.-deploy
-[10]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infra-operator
-[11]: https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-k8s-metrics-adapter
+## Values
```

Comment:
I would provide some further info regarding globals in general, what the user should expect when setting them.

Like:
 - no support for logging
 - in nri-kubernetes, local values are present for node selectors and hostnetwork

Reply:
I added another entry on top of values to state that.

Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)